docs/content/direct/examples.md

```diff
@@ -14,6 +14,15 @@ See [pre-reqs](pre-reqs.md).
 
 The following steps establish an initial state used in the examples below.
 
+1. You may want to `set -x` in your shell so that any failures in the setup or usage scenarios are not lost.
+
+1. If you ran through these scenarios previously then you will need to do a bit of cleanup first. Remove your kubeconfig contexts that were left behind and will get in the way of context rename commands to come.
+
+   ```shell
+   kubectl config delete-context cluster1
+   kubectl config delete-context cluster2
+   ```
+
```

Comment:
looking in e2e tests cleanup, it seems that it cleans more than these two contexts. it also removed the kind clusters - https://github.com/kubestellar/kubestellar/blob/main/test/e2e/common/cleanup.sh#L24-#L28 
(a) don't we want to clean the clusters as well?
(b) do we want to point to the cleanup script instead of writing the same here?

Reply:
I'm not sure we need to do any cleaning on the examples. For the user it should be enough to say "you should start from a clean environment"  , and maybe suggest to delete his whole environment. 
I agree with @nirrozenbaum - we can also point to our cleanup script as a reference.

Reply:
Good idea. Done.